cli/smartcontract/smart_contract.go

```diff
@@ -347,14 +347,15 @@ func contractCompile(ctx *cli.Context) error {
 
 	o := &compiler.Options{
 		Outfile: ctx.String("out"),
-		Debug:   ctx.Bool("verbose"),
```

Comment:
Well, maybe we should remove `Debug` from options at the same time? It's not used other than here.

Reply:
Done.